### PR TITLE
[#14] Use github actions for AWS deployment

### PR DESCRIPTION
**Pull request by diegomanuel**
This PR creates a new workflow **CI - Build release asset** at `build.yml`.

It runs when a new release is **published**, building the **release asset** and dispatching it to another repo, which runs the deployment with that release asset.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      - uses: actions/setup-elixir@v1.1.0
+        with:
+          otp-version: ${{ matrix.otp }}
+          elixir-version: ${{ matrix.elixir }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Install dependencies
+        run: |
```
Comment (pbrudnick) [inline .github/workflows/build.yml]: `env: prod` should be here?

Comment (pbrudnick) [reply .github/workflows/build.yml]: I mean:
```env:
          MIX_ENV: prod```

Comment (diegomanuel) [reply .github/workflows/build.yml]: Nopes, don't needed! It doesn't matter the `MIX_ENV` for this commands:
```
mix local.rebar --force
mix local.hex --force
mix archive.install --force .....phx_new.ez
mix deps.get
npm install
```

Comment (diegomanuel) [reply .github/workflows/build.yml]: The whole release build process is working fine!
New updates coming soon..  =]

Comment (pbrudnick) [reply .github/workflows/build.yml]: for the deps.get I mean, in the case you are using, in ex, `{:ex_doc, "~> 0.21", only: :dev, runtime: false},`

Comment (diegomanuel) [reply .github/workflows/build.yml]: Got it, you are right! It's a little improvement, will add it

**Comment by diegomanuel** [comment]
Already tested and works like a charm  =]

**Pull request merged**